posts/2022-09-22-Rust-1.64.0.md

```diff
@@ -209,6 +172,27 @@ These APIs are now usable in const contexts:
 
 - ...
 
+### Compatibility notes
+
+Rust 1.64 includes some changes that might prevent you from upgrading:
```

Comment:
Let's not use the phrasing "prevent you from upgrading". I think these two changes can just stand on their own without a lead-in.

Reply:
Good call.